doc/source/conf.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+#
+# clmdoc documentation build configuration file, created by
+# sphinx-quickstart on Thu Feb 23 17:14:30 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+# Note that we need a specific version of sphinx_rtd_theme. This can be obtained with:
+# pip install git+https://github.com/esmci/sphinx_rtd_theme.git@version-dropdown-with-fixes
+import sphinx_rtd_theme
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.intersphinx',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.imgmath',
+    'sphinx.ext.githubpages']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'ctsm'
+copyright = u'2020, UCAR'
+author = u''
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = u'CLM5.0'
+# The full version, including alpha/beta/rc tags.
+release = u'release-clm5.0.32'
```

Comment:
I wasn't sure what to do about `release`. What I currently have (release-clm5.0.32) follows what used to be in place in the ctsm-docs repo, but this doesn't seem ideal in terms of maintainability.

Reply:
Seems like instead of a specific release tag, we should point to the release branch. When there is a CESM release we might change it to the specific tag for that release.

so now it would be...

release-clm5.0

When a CESM release happens we'd update it to the specific version in that release. Or you could have it point to the latest release version that was in a CESM release which would be release-clm5.0.30.